README.md

```diff
@@ -72,47 +72,61 @@ You can define `Response` objects with `status`, `description`(optional)  a
 `header`(optional) and a `Resource`/`ListResource` object as `body` (optionally)...
 
 ```python
-ok_response = Response(
+list_response = Response(
     status=200, description='Here is your orders list.', body=list_resource
 )
 
-error_response = Response(status=400, resource=error_resource, headers=[('Content-type': 'application-json')])
+detail_response = Response(
+    status=200, description='Here is your orders list.', body=resource
+)
+
+error_response = Response(status=404, resource=error_resource, headers=[('Content-type': 'application-json')])
 ```
 
 ... and `Request` objects with `verb`, `description`, `header`(optional) and a `Resource`/`ListResource`
 object as `payload`.
 
 ```python
-request = Request(verb=verbs.GET, payload=resource)
+get_request = Request(verb=verbs.GET)
 ```
 
 An `Action` groups your request and a list of responses for an specified action
```

Comment:
An `Action` groups your request and a list of responses for (a) specified action passed in (the) description parameter. :lipstick: